crates/cast/src/starknet_commands/account/add.rs

```diff
@@ -60,7 +64,12 @@ pub async fn add(
     provider: &JsonRpcClient<HttpTransport>,
     add: &Add,
 ) -> Result<AccountAddResponse> {
-    let private_key = &SigningKey::from_secret_scalar(add.private_key);
+    let private_key = match &add.private_key_file_path {
+        Some(file_path) => get_private_key_from_file(file_path)
+            .with_context(|| format!("Failed to obtain private key from the file {file_path}"))?,
+        None => add.private_key.unwrap(),
```

Comment:
Let's generally obstain from unwrap-ing in prod code. The issue here is that we do not know if private_key is not None, and if it is this will panic - which ewill abruptly terminate the program.

Unwraping may be ok if you are sure about the Result type (or in tests).

Reply:
Thanks! It this case it actually cannot be None as clap would throw an error earlier if both `private-key` and `private-key-file` are not provided, but I get the point.